content/en/observability_pipelines/setup/datadog_with_archiving.md

```diff
@@ -0,0 +1,529 @@
+  ## your logs can be miscategorized at intake.
+  logs_remove_wrong_level:
+    type: remap
+    inputs:
+      - logs_parse_ddtags
+    source: |
+      del(.status)
+
+  ## This is a placeholder for your own remap (or other transform)
+  ## steps with tags set up. Datadog recommends these tag assignments.
+  ## They show which data has been moved over to OP and what still needs
+  ## to be moved.
+  LOGS_YOUR_STEPS:
+    type: remap
+    inputs:
+      - logs_remove_wrong_level
+    source: |
+      .ddtags.sender = "observability_pipelines_worker"
+      .ddtags.opw_aggregator = get_hostname!()
+
+  ## Before sending data to the logs intake, you must re-encode the
+  ## tags into the expected format, so that it appears as if the Agent is
+  ## sending it directly.
+  logs_finish_ddtags:
+    type: remap
+    inputs:
+      - LOGS_YOUR_STEPS
+    source: |
+      .ddtags = encode_key_value!(.ddtags, key_value_delimiter: ":", field_delimiter: ",")
+
+  metrics_add_dd_tags:
+    type: remap
+    inputs:
+      - datadog_agent.metrics
+    source: |
+      .tags.sender = "observability_pipelines_worker"
+      .tags.opw_aggregator = get_hostname!()
+
+## This buffer configuration is split into the following, totaling the 288GB
+## provisioned automatically by the Terraform module:
+## - 240GB buffer for logs
+## - 48GB buffer for metrics
+##
+## This should work for the vast majority of OP Worker deployments and should rarely
+## need to be adjusted. If you do change it, be sure to update the `ebs-drive-size-gb`
+## parameter.
+sinks:
+  datadog_logs:
+    type: datadog_logs
+    inputs:
+      - logs_finish_ddtags
+    default_api_key: "$${DD_API_KEY}"
+    compression: gzip
+    buffer:
+       type: disk
+       max_size: 257698037760
+  datadog_metrics:
+    type: datadog_metrics
+    inputs:
+      - metrics_add_dd_tags
+    default_api_key: "$${DD_API_KEY}"
+    buffer:
+      type: disk
+      max_size: 51539607552
+  ## This sink writes logs to an S3 bucket of your choice, in a format that Datadog
+  ## can rehydrate from. You need to fill in the ${DD_ARCHIVES_BUCKET}
+  ## and ${DD_ARCHIVES_REGION} parameters to fit your S3 configuration.
+  datadog_archives:
+    type: datadog_archives
+    inputs:
+      - logs_finish_ddtags
+    service: aws_s3
+    bucket: ${DD_ARCHIVES_BUCKET}
+    aws_s3:
+      storage_class: "STANDARD"
+      region: "${DD_ARCHIVES_REGION}"
+EOT
+}
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+### Load balancing
+
+{{< tabs >}}
+{{% tab "Docker" %}}
+Production-oriented setup is not included in the Docker instructions. Instead, refer to your company's standards for load balancing in containerized environments. If you are testing on your local machine, configuring a load balancer is unnecessary.
+{{% /tab %}}
+{{% tab "AWS EKS" %}}
+Use the load balancers provided by your cloud provider.
+They adjust based on autoscaling events that the default Helm setup is configured for. The load balancers are internal-facing,
```

Comment:
What is `they` referencing, the load balancers or the cloud provider?

Reply:
Load balancer, updated it.